app/safe/cmd/main.go

```diff
@@ -21,6 +21,8 @@ import (
 func main() {
 	id := "VSECMSAFE"
 
+	bootstrap.ValidateEnvironment()
```

Comment:
I think validating env vars explicitly is better.

Using `init()` there could be hard to track down; whereas here, we explicitly state that we are validating the environment the first thing in the initialization process.

Clean is better than clever.